editor/animation_track_editor.cpp

```diff
@@ -1632,6 +1637,45 @@ void AnimationTimelineEdit::set_zoom(Range *p_zoom) {
 	zoom->connect("value_changed", callable_mp(this, &AnimationTimelineEdit::_zoom_changed));
 }
 
+
+void AnimationTimelineEdit::auto_fit() {
+
+	if (!animation.is_valid()) {
+		return;
+	}
+
+	float anim_length = animation->get_length();
+
+	int timeline_width_pixels = get_size().width - get_buttons_width() - get_name_limit();
+
+	//I want a little buffer at the end... (5% looks nice and we should keep some space for the bezier handles)
```

Comment:
```suggestion
	// I want a little buffer at the end... (5% looks nice and we should keep some space for the bezier handles).
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).